hook.go

```diff
@@ -212,7 +212,7 @@ func (h *Hook) getCredentials() *credentials {
 				APIToken:      queryString("apitoken"),
 				APIURL:        queryString("apiurl"),
 				SkipErrors:    queryString("skiperrors") == "true",
-				Location:      queryString("location"),
+				Location:      queryString("networkzone"),
```

Comment:
Can you also change the name for the object field to `NetworkZone`? Not necessary but for consistency.

Reply:
done